src/app/core.clj

```diff
@@ -37,7 +40,7 @@
   (println banner)
   (System/exit 1))
 
-(defn info [opts msg]
+(defn info [msg]
   (println " [info] " msg))
 
 (defn self-exclusive-modes?
```

Comment:
I know it has not been changed in this PR but maybe we could rename `self-exclusive-modes?` to `conflicting-options?` or `multiple-modes?`